aws/emr/cdo-logs/script/elb_mapper.rb

```diff
@@ -84,13 +84,17 @@
   date = line[0..9]
 
   tutorials.each do |t|
-    if (line.include? "begin" + "\/" + t) ||
-      (line.include? "begin" + "_" + t) ||
-      (line.include? "443" + "\/" + t)
-      # Using LONG_VALUE_SUM instructs hadoop's streaming aggregate class how to
-      # aggregate. Using the date and tutorial as the key gives breakdowns by
-      # day and tutorial.
-      puts LONG_VALUE_SUM + date + " " + t + "\t" + "1"
+    # Using LONG_VALUE_SUM instructs hadoop's streaming aggregate class how to
+    # aggregate. Using the date and tutorial as the key gives breakdowns by
```

Comment:
Could ya update this comment to mention that we're using date, tutorial, and type of metric as the key?